yellowbrick/classifier/boundaries.py

```diff
@@ -0,0 +1,457 @@
+
+        colors : string or matplotlib cmap, default: yellowbrick palette 'set1'
+            Sets the color palette of the visualizer
+
+        classes : a list of class names for the legend, default: None
+            If classes is None and a y value is passed to fit then the classes
+            are selected from the target vector.
+
+        features : list of strings, default: None
+            The names of the features or columns
+
+        show_scatter : boolean, default: True
+            If boolean is True, then a scatter plot with points will be drawn
+            on top of the decision boundary graph
+
+        step_size : float percentage, default: 0.0025
+            Determines the step size for creating the numpy meshgrid that will
+            later become the foundation of the decision boundary graph. The
+            default value of 0.0025 means that the step size for constructing
+            the meshgrid will be 0.25%% of differenes of the max and min of x
+            and y for each feature.
+
+        markers : iterable of strings, default: ,od*vh+
+            Matplotlib style markers for points on the scatter plot points
+
+        pcolormesh_alpha : float, default: 0.8
+            Sets the alpha transparency for the meshgrid of model boundaries
+
+        scatter_alpha : float, default: 1.0
+            Sets the alpha transparency for the scatter plot points
+
+        title : string, default: stringified feature_one and feature_two
+            Sets the title of the visualization
+
+        kwargs : keyword arguments passed to the super class.
+
+        These parameters can be influenced later on in the visualization
+        process, but can and should be set as early as possible.
+
+    """
+
+    def __init__(self,
+                 model,
+                 x=None,
+                 y=None,
+                 colors=None,
+                 classes=None,
+                 features=None,
+                 show_scatter=True,
+                 step_size=0.0025,
+                 markers=None,
+                 pcolormesh_alpha=0.8,
+                 scatter_alpha=1.0,
+                 title=None,
+                 **kwargs):
+        """
+        Pass in a unfitted model to generate a decision boundaries
+        visualization.
+        """
+        super(DecisionBoundariesVisualizer, self).__init__(self)
+
+        self.x = x
+        self.y = y
+        self.colors = colors if colors is not None else PALETTES['set1']
+        self.classes_ = classes
+        self.features_ = features
+        self.estimator = model
+        self.name = get_model_name(self.estimator)
+        self.show_scatter = show_scatter
+        self.step_size = step_size
+        self.markers = itertools.cycle(
+            kwargs.pop('markers', (',', 'o', 'd', '*', 'v', 'h', '+')))
+        self.pcolormesh_alpha = pcolormesh_alpha
+        self.scatter_alpha = scatter_alpha
+        self.title = title
```

Comment:
Note that title is handled in the super class, actually in the very base class Visualizer, and should be omitted here to ensure that `set_title()` works.